doc/modules/gaussian_process.rst

```diff
@@ -154,6 +154,10 @@ predictions. Moreover, note that :class:`GaussianProcessClassifier` does not
 as discussed above is based on solving several binary classification tasks
 internally, which are combined using one-versus-rest or one-versus-one.
 
+.. rubric:: Examples
+
+   :ref:`sphx_glr_auto_examples_classification_plot_classification_probability.py`
```

Comment:
The indentation is not correct. Also, for consistency, can you include the star at the front (like in the other references)? This will start a list. It's not really necessary here because there's only one example but it's better to adhere to a certain format.

Reply:
Complete